packages/pods-core/src/types.ts

```diff
@@ -20,6 +20,11 @@ export type PodItemV4 = {
 
 export type PodKind = "import" | "export" | "publish";
 
+export enum Source {
```

Comment:
Rather than having this enum, let's just add a `port` parameter to the constructor of `GoogleDocsExportPodV2`.  Because it's really the _port_ that affects the behavior of the gdoc pod rather than whether it's plugin vs cli.  This helps better expose the true behavior dependency for configuration.